### ISSUE TITLE
Bass fret events within programming wheel

### ISSUE THREAD

**Issue by iansedano** [wintergatan-community/vmmx-schema#13]
I am not sure if I have got this wrong but it seems that the BassDropEvent is used to populate the program only.
Within the BassDropEvent there is an option to specify the fret that is held down.
Does this mean that the frets that are held down are "hard coded" into the programming wheel loop and can't be changed during the performance?
If so, I believe this to be a mistake, and that the fretting events should be part of the performance.

![200529_154527_chrome](https://user-images.githubusercontent.com/38072365/83266675-72816a00-a1c3-11ea-966f-be807a604b28.png)

This graphic represents the bass line for the original marble machine song. Each separator is half a revolution of the main programming wheel. It changes from one revolution to the other which is what gives the performance depth and movement.

**Comment by mozi-h** [comment]
The fretting is "baked" into the BassDropEvent to allow for auto-played bass, without the need to constantly click them during playback.
The plan is to always allow manual fretting to achieve a different note. You may also just chose to not "bake" in any bass and leave them in their default auto-fret of nothing.

**Comment by iansedano** [comment]
* I am not seeing where the fret events are in the performance part of the schema, only the capo. Am I missing something?

* It will be possible to "write in" performance events, right? Fretted notes for example, or will that have to be manual?

**Comment by mozi-h** [comment]
- The "baked" frets are part of the `BassDropEvent`:
```typescript
export interface BassDropEvent {
	kind: "bass";
	string: BassString;
	fret: number;
}
```

But turns out there actually is a problem hiding! See PR #14.

**Comment by mozi-h** [comment]
I think this is resolved as of #14
Scream at me if that’s not the case
